epub33/rs/index.html

```diff
@@ -2066,8 +2045,9 @@ <h2>Processing structural semantics</h2>
 		<section id="sec-vocab-assoc">
 			<h2>Vocabulary association mechanisms</h2>
 
-			<p id="confreq-rs-vocab-assoc" class="support">Reading systems MUST support <a
-					data-cite="epub-33#sec-vocab-assoc">vocabulary association mechanisms</a> [[epub-33]].</p>
+			<p id="confreq-rs-vocab-assoc" class="support">Reading systems support for <a
+					data-cite="epub-33#sec-vocab-assoc">vocabulary association mechanisms</a> [[epub-33]] is
+				OPTIONAL.</p>
```

Comment:
```suggestion
			<p id="confreq-rs-vocab-assoc" class="support">Reading systems MAY support <a
					data-cite="epub-33#sec-vocab-assoc">vocabulary association mechanisms</a> [[epub-33]] in [=EPUB content documents=].</p>
```

(Just to be consistent with similar notices elsewhere. Besides, I believe the current language was not correct.)